go.mod

```diff
@@ -8,15 +8,17 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.1.0
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.10.2
```

Comment:
Are we using both versions of this now? I don't see any code changes.

Reply:
We're going to need it to bring in the new `svc` but that's blocked by the `buildenv` PR. I'm not sure why `go mod` is bringing this in now, I don't actually see it used: `(main module does not need package github.com/grpc-ecosystem/grpc-gateway/v2)`

Reply:
I had a dirty working dir. I deleted some left over files and now this import is gone.